starter-code/index.html

```diff
@@ -22,6 +23,7 @@
     </section>
     <nav class="main-nav">
       <!-- NOTE: (Stretch Goal) WE WILL DO THIS IN LECTURE 02 - Insert here a 'hamburger' menu icon font (or find your own img). Using a div is okay! -->
+      <div class="icon-menu"><img src="../hamburger.png" id="hamburger"></div>
```

Comment:
This was a stretch goal, so no biggie, but the goal was to incorporate a glyph icon (they are included in the starter-code styles as fonts), not an image. I like your hamburger image, though! 🍔